scrapinghub/exceptions.py

```diff
@@ -62,6 +63,11 @@ def wrapped(*args, **kwargs):
             elif 400 <= status_code < 500:
                 raise ScrapinghubAPIError(http_error=exc)
             raise
+        except APIError as exc:
+            msg = exc.args[0]
+            if 'format must be either json or jl' in msg:
```

Comment:
@chekunkov Did you mean something like that https://github.com/scrapinghub/python-scrapinghub/pull/47? 
(Moved the changes to a separate PR for simplicity.)